test/src/main/java/org/apache/accumulo/test/GetManagerStats.java

```diff
@@ -148,11 +148,11 @@ public static void main(String[] args) throws Exception {
     }
   }
 
-  private static void out(int indent, String string, Object... args) {
+  private static void out(int indent, String fmtString, Object... args) {
     for (int i = 0; i < indent; i++) {
       System.out.print(" ");
     }
-    System.out.println(String.format(string, args));
+    System.out.printf((fmtString) + "%n", args);
```

Comment:
```suggestion
    System.out.printf(fmtString + "%n", args);
```

Reply:
I just accepted the IDE suggested changes - looking at your suggestion, I can't find where adding the %n is necessary to add to the calling fmt string.

Reply:
Update with proper newlines added in #76fa72c862